css/store.css

```diff
@@ -0,0 +1,261 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  outline: 0;
+  border: 0;
+}
+
+h1 {
```

Comment:
Many of these styles are duplicated from style.css, can you think of a way to reduce this duplication? Maybe splitting into multiple files?

Reply:
Thank you Barath. I just copied the Header and the Footer styles from the Styles.css. Is there another way for that? 🤔 Thank you

Reply:
Hey @Ali-Jahankah, one idea is to split some of the CSS out into a common.css file which can get used on both pages - it can contain CSS which is common to the site.

Reply:
@moneyinthesky Thank you I just did it.